### DISCUSSION TITLE
Getting Deprecated issue on pc, application runs in browser but will not let me hit enter for request

### DISCUSSION THREAD

**Discussion by CCesternino** [jas3333/GPT-Helper#7] [Q&A]
> GPT3Chat@0.1.0 start
> react-scripts start

(node:16448) [DEP_WEBPACK_DEV_SERVER_ON_AFTER_SETUP_MIDDLEWARE] DeprecationWarning: 'onAfterSetupMiddleware' option is deprecated. Please use the 'setupMiddlewares' option.
(Use `node --trace-deprecation ...` to show where the warning was created)
(node:16448) [DEP_WEBPACK_DEV_SERVER_ON_BEFORE_SETUP_MIDDLEWARE] DeprecationWarning: 'onBeforeSetupMiddleware' option is deprecated. Please use the 'setupMiddlewares' option.
Starting the development server...

Compiled successfully!

You can now view GPT3Chat in the browser.

**Comment by jas3333** [comment]
Most likely an issue with your API key. Make sure your `.env` file is named just `.env` inside the client folder. Variable inside the file should be  `REACT_APP_OPENAI_KEY=yourkeywithoutquotes`. You can also open up the dev tools in the browser and check the console for any errors.
